content/doc/book/managing/plugins.adoc

```diff
@@ -387,3 +387,43 @@ image::managing/hover-for-implied-dependencies.png["Hover over the uninstall but
 
 The implied dependency can be removed by releasing a new version of the plugin that depends on a newer minimum Jenkins core. 
 The link:/doc/developer/tutorial-improve/[Improve a plugin tutorial] provides steps that can assist plugin developers as they update the plugin to link:/doc/developer/tutorial-improve/update-base-jenkins-version/[depend on a newer minimum Jenkins core].
+
+[[plugin-health-score]]
+== What is the health score?
+
+There is a metric available next to the plugins in the Plugin Manager called "Health Score".
+
+image::managing/plugin-manager-available-health-score.png["Plugin Health Score in the Plugin Manager available listing", role=center]
+
+You can find the same metric in the available update and the installed page of the plugin manager.
+
+NOTE: The score value is always based on the latest available data.
+This means that there is no score of a specific release of a plugin.
+What is available on the installed and updates page of the plugin manager does not stand for the version already installed.
+
+This metric is coming from link:https://github.com/jenkins-infra/plugin-health-scoring[Plugin Health Scoring].
+
+This service is gathering data from each plugin.
+
+The data can be related to the source code of the plugin, the content of the plugin repository or external metrics about the plugin such as number of open pull requests, etc.
```

Comment:
```suggestion
The data can be related to the source code of the plugin, the content of the plugin repository, or external metrics about the plugin such as number of open pull requests.
```
Small suggestion to add oxford comma and remove use of `etc.` since there is only one example of the external metrics.